readme: updates

illwieckz opened the pull request:
More readme updates.

I also squashed the commit for an unsolved thread from https://github.com/Unvanquished/Unvanquished/pull/2087

necessarily-equal commented on README.md:
```diff
@@ -89,9 +85,23 @@ git submodule update --init --recursive
-Instead of `-j4` you can use `-jN` where `N` is your number of CPU cores to distribute compilation on them. Linux systems usually provide an handy `nproc` tool that tells the number of CPU core so you can just do `-j$(nproc)` to use all available cores.
+If you don't have the assets, you can download them first. You can download an universal zip from the [download page](https://unvanquished.net/download/), here is the [link to the latest version](https://unvanquished.net/download/zip). The `pkg/` folder would have to be stored next to your `daemon` binary.
+
+Otherwise you can use the package downloader script. This script can use `aria2c`, `curl` or `wget`, `aria2c` is recommended.
```
```suggestion
Otherwise you can use the package downloader script. This script can use `aria2c`, `curl` or `wget`. `aria2c` is recommended.
```

illwieckz replied:
Done

necessarily-equal commented on README.md:
```diff
@@ -132,87 +142,127 @@ cmake --build build -- -j4
+
+You can use `-pakpath` to load base assets from a `pkg/` folder stored somewhere else (see above).
+
+For debugging purpose, you can run your mod as dll:
```
```suggestion
For debugging purpose, you can run your mod as a shared library:
```

illwieckz replied:
Done

necessarily-equal commented on README.md:
````diff
@@ -132,87 +142,127 @@ cmake --build build -- -j4
+You then start a game server with your custom mod this way:
+
+```sh
+./daemonded set fs_extrapaks mod-custom +map plat23
````
```suggestion
./daemonded -set fs_extrapaks mod-custom +map plat23
```
(or maybe `+set` works too)

illwieckz replied:
Done

necessarily-equal commented:
GitHub is missing this feature GitLab has where you can diff with the previous version of the PR you reviewed, and I guess you didn't change that much so I don't think I'll be reviewing it again. I think you can merge it when it's ready™

illwieckz commented:
Yeah, I addressed the issues you pointed out and reworded some wordy words in most concise manner.

Something I'm looking for a comment: I tried to add some ℹ️ and 💡️ emojis to mark some “ℹ️  Notable info” and some “💡️ Clever trick”. What do you think about it?

DolceTriade commented:
lgtm
